Makefile

```diff
@@ -204,6 +204,8 @@ ccache/em++:
 
 $(CPYTHONLIB): emsdk/emsdk/.complete ccache/emcc ccache/em++
 	make -C $(CPYTHONROOT)
+	python -m pip install -e .
+	"$(HOSTPYTHONROOT)/bin/python3" -m pip install -e .
```

Comment:
We install the `pyodide_build` package both in the regular python and the host python. It just creates a symlink, so it's not too costly. I'm not sure if this is the right place to put it though.

For the system python, this probably assumes that we are in some kind of virtualenv though and this might fail with insufficient permissions if pip attempts to install it somewhere under `/usr/lib`.  The alternative could be to ask users install it explicitly before running make.